v3/tasks/20241205-v3-vault-explorer/readme.md

```diff
@@ -0,0 +1,8 @@
+# 2024-12-05 - Vault Explorer
+
+Deployment of `VaultExplorer`, a "window" on the Vault that provides easy access to view functions across all Vault contracts for off-chain utilities like Etherscan. Since the Vault implements the Proxy pattern to fit within the EVM bytecode requirements, most functions are on extension contracts that cannot be called directly. The `VaultExplorer` is a wrapper contract that exposes all permissionless view functions (plus `collectAggregateFees`).
```

Comment:
```suggestion
Deployment of `VaultExplorer`, a "window" on the V3 Vault that provides easy access to view functions across all Vault contracts for off-chain utilities like Etherscan. Since the Vault implements the Proxy pattern to fit within the EVM bytecode requirements, most functions are on extension contracts that cannot be called directly. The `VaultExplorer` is a wrapper contract that exposes all permissionless view functions (plus `collectAggregateFees`).
```

I think it's nice to mention that VaultExplorer is just for V3 vault